tokens/contracts/access/MinterAccessControl.sol

```diff
@@ -10,40 +10,18 @@ contract MinterAccessControl is Initializable, OwnableUpgradeable {
     using EnumerableSetUpgradeable for EnumerableSetUpgradeable.AddressSet;
 
     EnumerableSetUpgradeable.AddressSet private _minters;
-    bool public minterAccessControlEnabled;
-
-    event MinterAccessControlEnabled();
-    event MinterAccessControlDisabled();
+    
     event MinterGranted(address indexed account);
     event MinterRevoked(address indexed account);
 
-    function __MinterAccessControl_init() internal initializer {
+    function __MinterAccessControl_init() external initializer {
```

Comment:
I'm not sure what's this `external` is for. It feels like this should not be something that we want to expose, is it?
I see this is used in tests but should it be done for production?